backend/api-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,11 @@
+spring:
+  datasource:
+    url: jdbc:mysql://localhost:3306/boutique_hotel
+    username: SET_ME
+    password: SET_ME
+    driver-class-name: com.mysql.cj.jdbc.Driver
+  jpa:
+    database-platform: org.hibernate.dialect.MySQL8Dialect
+    hibernate:
+      ddl-auto: update
+    show-sql: true
```

Comment:
open-in-view: false

as open-in-view most probably should be disabled: https://stackoverflow.com/questions/30549489/what-is-this-spring-jpa-open-in-view-true-property-in-spring-boot

Warning when starting the server: spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning